ui/app/styles/core.scss

```diff
@@ -23,7 +23,7 @@
 @import './core/charts';
 @import './core/checkbox-and-radio';
 @import './core/columns';
-@import './core/container';
```

Comment:
I use singular only when that's the only class being defined. When it's container type classes, then I use plural of the word. I'll document this in the patterns doc.